splunk-otel-android/src/main/java/com/splunk/rum/SplunkRumBuilder.java

```diff
@@ -368,7 +381,11 @@ boolean isReactNativeSupportEnabled() {
         return configFlags.isReactNativeSupportEnabled();
     }
 
-    boolean isBackgroundTaskReportingDisabled() {
-        return !configFlags.isBackgroundTaskInstrumentationEnabled();
+    boolean isSubprocessInstrumentationEnabled() {
+        return !configFlags.isSubprocessInstrumentationEnabled();
+    }
```

Comment:
I think this method is named wrong now. Because it's inverted, it should be disabled.
```suggestion
    boolean isSubprocessInstrumentationDisabled() {
        return !configFlags.isSubprocessInstrumentationEnabled();
    }
```